addons/portal_rating/static/src/js/portal_rating_composer.js

```diff
@@ -91,6 +91,10 @@ const RatingPopupComposer = publicWidget.Widget.extend({
                 _t('Review') : this.options.default_message_id ?
                 _t('Edit Review') : _t('Add Review')
             );
+            this.$('.o_portal_chatter_composer_btn').text(
+                this.options.default_message_id ?
+                _t('Update review') : _t('Post review')
+            );
```

Comment:
We're trying to avoid introducing new jquery usage in Odoo :smile: 
Could you change the text without using jquery (with `querySelector` and maybe `textContent`) ?